packages/example-playground/src/examples/x.counter/source.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, TSXAir } from '../../framework';
+export const StatefulComp = TSXAir((props: { seconds: number }) => {
+
+    let counter = 0;
+    let clickeCounter = 0;
+    let label = '';
```

Comment:
This is unclear - why is this line not executed upon change, and the following line is.
while technically possible, It creates an inconsistent experience for the developer.
Explicit syntax (possibly a lifecycle) is IMHO easier to understand  and work with. 
Less magic would be better here. 
An interesting point to discuss is how the debugging experience going to be here

Reply:
checkout this alternatives.

also the compiled is very different